### DISCUSSION TITLE
Websocket protocal

### DISCUSSION THREAD

**Discussion by smehta118kjse** [graphql-java-generator/graphql-maven-plugin-project#169] [General]
I am using this repo to generate client stub code. Queries and mutations works fine.
1) For websockets does it support graphql-ws protocol ? I see it supports graphql-transport-ws . are these the same ?
2) The code is unable to init with the initial message with access token once the connection is established. I couldn't figure out and websocket is closed and after 30 seconds time out the process is terminated. Also, is there a way to control the timeout. Any pointers would be helpful. Tx

**Comment by etienne-sf** [comment]
Hello,

The plug-in supports the [graphql-transport-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md). 

I don't understand the sentence "The code is unable to init with the initial message with access token once the connection is established"

What access token are your talking about ?

Étienne
